superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -208,6 +211,28 @@ const ChangeDatasourceModal: FunctionComponent<ChangeDatasourceModalProps> = ({
     return data;
   };
 
+  const customGotoPage = (p: number) => {
+    SupersetClient.get({
+      endpoint: `/api/v1/dataset/?q=${rison.encode({
+        order_column: 'changed_on_delta_humanized',
+        order_direction: 'desc',
+        page: p - 1,
+        page_size: 20,
+      })}`,
+    }).then(resources => {
+      setPageIndex(p - 1);
+      setResourceCollection(resources.json.result);
+    });
+  };
+
+  const styleOverride = css`
+    .table-condensed {
```

Comment:
Since it's the `TableView` component is using this `className`, you could add the `css` prop to `TableView` component directly, rather than this ancestor. Then you don't have to have the class selector in the styles here, and the styles are closer to where they're used.

Reply:
Also, adding the styles via a `const` like this is OK, but it's also OK add them more directly as you've done elsewhere, with
```js
<Table View
  css={theme => css`
    /* styles here... */
  `
 ...
}
```

Reply:
I rather do it to the ancestor since tableView is being used by other components and I only want to change this style wit this particular modal component.